oteapi_dlite/strategies/oceanlab_influx_parser.py

```diff
@@ -89,9 +89,7 @@ class InfluxParseParseConfig(AttrDict):
 
     measurements: Annotated[
         list[dict],
-        Field(
-            description="list of measurements containing measurement and field values as list of dictionary"
-        ),
+        Field(description="Measurement and field values as list of dictionary"),
```

Comment:
I'm guessing this is something left over from the other commits/PR?

Reply:
yes